spotbugs/src/main/java/edu/umd/cs/findbugs/visitclass/AnnotationVisitor.java

```diff
@@ -143,8 +143,8 @@ public void visitSyntheticParameterAnnotation(int p, boolean runtimeVisible) {
         }
         return values;
     }
-
-
+    
+    
```

Comment:
Should we just delete a line here?

Reply:
I do not think so. It is fine like this.
IMO, To remove comment-out lines is better than to delete a line.

I would like to separate issue.